modules/logwriter/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "function" {
+  description = "Lambda Function ARN"
+  value       = length(module.subscriber) > 0 ? module.subscriber[0].function_arn : ""
```

Comment:
It might be better to just return the subscriber:
```
output "subscriber" {
  description = "Subscriber module"
  value       = one(module.subscriber)
}
```